### ISSUE TITLE
sed: couldn't open temporary file - Permission denied

### ISSUE THREAD

**Issue by Mik4sa** [mko-x/docker-clamav#93]
**Describe the bug**
Running the (latest) version with environment variables to replace specific values in the config/s results into following error:
```
sed: couldn't open temporary file /etc/clamav/sedqlfOoV: Permission denied
sed: couldn't open temporary file /etc/clamav/sedQ1LUtW: Permission denied
```

**To Reproduce**
1. Run a container with the latest tag and set a value via environment variables. For example:
`docker run -e "CLAMD_CONF_StreamMaxLength=101M" mkodockx/docker-clamav`

**Expected behavior**
No errors, successful start and altered StreamMaxLength value is used.

**Host (please complete the following information):**
 - OS: Azure Container Instances (container group), but also on a windows host (Version 2004 (Build 19041.662)) with wsl2

**Image (please complete the following information):**
 - Tag: latest
 - Configuration: Environment variable `CLAMD_CONF_StreamMaxLength` set to `101M`

**Comment by mko-x** [comment]
Could you please provide the permissions for /etc/clamav within the image after the error occured?

This image was not tested for setting clamd environment variables via docker in Azure. Did you try setting a custom conf file instead?

**Comment by Mik4sa** [comment]
Since the default settings where fine (except StreamMax), I didn't tested a custom file. I actually replaced your image with my own, since I needed a fast solution to this.
So actually this no longer is a problem to me.

I used your image before and I never needed to assign any special permissions. Though, I can try... It now works again with any modifications. I guess it was: https://github.com/mko-x/docker-clamav/commit/5e5d934f2c4c997325ac42b1b3a2c72132df00dc
Thx @ericmason
